boskos/resources.json

```diff
@@ -1,5 +1,822 @@
 [
-{"name": "FAKE_PROJECT_1", "type": "project", "state": "free"},
-{"name": "FAKE_PROJECT_2", "type": "project", "state": "free"},
-{"name": "FAKE_PROJECT_3", "type": "project", "state": "free"}
+  {
```

Comment:
Once we have boskos managing 200 jobs, how do you intend to add the 201st one without causing an outage? If the config and state formats are the same this seems dangerous, we're likely to reset everything as free and then cause a bunch of problems.



Change the type from project to gke-project, gce-project, large-project, etc.